MANIFEST.in

```diff
@@ -1,3 +1,2 @@
 include *.rst
-include requirements2.txt
```

Comment:
Since we drop py2 support, the requirements3.txt should be renamed back to requirements.txt

Reply:
will do this in another iteration.

Reply:
done.